Packs/Base/Scripts/GetErrors/GetErrors_test.py

```diff
@@ -0,0 +1,49 @@
+from GetErrors import get_errors
+
+
+ERROR_ENTRY_1 = [{'Contents': 'This is the error message 1', 'Type': 4}]
```

Comment:
`4` is `entryTypes['error']`?
If yes - Why not use the same value?

Reply:
you can use the enum from CSP